src/dlgProfilePreferences.cpp

```diff
@@ -104,6 +104,26 @@ dlgProfilePreferences::dlgProfilePreferences(QWidget* pF, Host* pHost)
         comboBox_toolBarVisibility->setCurrentIndex(2);
     }
 
+    // Set the properties of the log options
+    lineEdit_logFileFolder->setToolTip(QStringLiteral("<html><head/><body>%1</body></html>")
+                                               .arg(tr("<p>Shows the location which will be used to store log files (either plain text or HTML format)."
```

Comment:
Way too wordy and kind of overbearing ("stupid, you can't change the text here, look at the buttons on the right")

How about this:

```
"<p>Location which will be used to store log files - matching logs will be appended to.</p>")
```

Reply:
That sounds great. Just pushed commit 53c76af, which should address that.